packages/kit/src/exports/index.js

```diff
@@ -84,6 +84,14 @@ export function isHttpError(e, status) {
 /**
  * Redirect a request. When called during request handling, SvelteKit will return a redirect response.
  * Make sure you're not catching the thrown redirect, which would prevent SvelteKit from handling it.
+ *
+ * Most common status codes:
+ *  * `303 See Other`: redirect as a `GET` request
+ *  * `307 Temporary Redirect`: redirect will keep the request method
+ *  * `308 Permanent Redirect`: redirect will keep the request method, SEO will be transferred to the new page
```

Comment:
I'm not entirely confident that 307 doesn't have some SEO implications. I'd simplify it to remove that

Reply:
> Search engines receiving this response will not attribute links to the original URL to the new resource, meaning no [SEO](https://developer.mozilla.org/en-US/docs/Glossary/SEO) value is transferred to the new URL.

[Per MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/307), although this is not a guarantee of any kind on Google's end